### DISCUSSION TITLE
The package won't install

### DISCUSSION THREAD

**Discussion by tomhopkins** [software-mansion/react-native-gesture-handler#3148] [Q&A]
When I run npm i react-native-gesture-handler

I get

npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/react-native-gesture-handler/ReanimatedSwipeable.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

**Comment by m-bert** [comment]
Hi @tomhopkins! Do you use latest version of `npm`? If not, run

```bash
npm install -g npm 
```

If that doesn't help, you can try to delete `node_modules`, `package-lock.json` and then clean cache with

```
npm cache clean --force
```

After this you should be able to install Gesture Handler.
